contrib/mesos/pkg/scheduler/scheduler.go

```diff
@@ -923,6 +924,10 @@ func (ks *KubernetesScheduler) recoverTasks() error {
 		ks.slaves.checkAndAdd(slaveId, t.Offer.Host())
 	}
 	for _, pod := range podList.Items {
+		if _, isMirrorPod := pod.Annotations[kubelet.ConfigMirrorAnnotationKey]; isMirrorPod {
+			// mirrored pods are never reconciled
```

Comment:
got it, will clarify

On Mon, Sep 14, 2015 at 9:24 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/scheduler/scheduler.go
> <https://github.com/kubernetes/kubernetes/pull/13918#discussion_r39392150>
> :
>
> > @@ -923,6 +924,10 @@ func (ks *KubernetesScheduler) recoverTasks() error {
> >  		ks.slaves.checkAndAdd(slaveId, t.Offer.Host())
> >  	}
> >  	for _, pod := range podList.Items {
> > +		if _, isMirrorPod := pod.Annotations[kubelet.ConfigMirrorAnnotationKey]; isMirrorPod {
> > +			// mirrored pods are never reconciled
>
> I meant the comment. it doesn't add anything to what is immediate in the
> code line above. The "why" would be helpful.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13918/files#r39392150>.
>